lib/ansible/module_utils/network/junos/junos.py

```diff
@@ -37,6 +37,7 @@
 JSON_ACTIONS = frozenset(['merge', 'override', 'update'])
 FORMATS = frozenset(['xml', 'text', 'json'])
 CONFIG_FORMATS = frozenset(['xml', 'text', 'json', 'set'])
+DEFAULT_COMMENT = 'configured by junos_config'
```

Comment:
What is the reason to add a default comment for all the modules? For declarative modules, this might cause idempotency related issues

Reply:
I add this default comment because of compatibility.
I move the function out from `junos_config`, so I need to keep the same output when someone using `junos_config` module in old version.
If you don't care about it, it's ok to change.